avalon/tools/projectmanager/widget.py

```diff
@@ -138,7 +138,8 @@ def preserve_selection(tree_view,
                 selection_model.select(index, flags)
 
             if current_index_value and value == current_index_value:
-                tree_view.setCurrentIndex(index)
+                selection_model.setCurrentIndex(index,
```

Comment:
Actually correctly set the current index - not entirely sure why this was needed anymore or where the behavior felt off. I believe on creating assets previously it would change the current index you were working on - this fixed that I believe.